redaction/redactor.go

```diff
@@ -134,10 +167,13 @@ func (redactor *Redactor) Write(input []byte) (int, error) {
 	// May lag behind cursor by up to the length of the longest search string
 	doneTo := 0
 
+	pretty.Println(redactor.table)
+
 	// For so long as the safe consumption index is inside the current input array
 	for cursor < len(input) {
 		ch := input[cursor]
-		skip := redactor.table[ch].skip
+		fmt.Printf("ch: %c\n", rune(ch))
+		skip := redactor.table.Skip(rune(ch))
```

Comment:
this solution doesn't work because of this line. the problem is that we store all of the skip values as a `map[rune]int`, but the redactor looks through its input byte-by-byte. the issue here is that when we call `ch := input[cursor]`, we're selecting the `cursor`th _byte_ from input, which might only represent some of the "character" that we're looking for.